Automattic/hidden-posts
# Use .pot-based instead of filter-based translation

Fixes #20 

**Note**

Use .pot-based instead of filter-based translation.

**Steps to test**

1. Check out this PR
2. Go to /wp-admin/options-general.php
3. Switch the site language to German
4. Go to /wp-admin/edit.php and ensure that the column titles `Visibility` and `Date` are translated to `Sichtbarkeit` and `Datum`

In addition, feel free to create your own .mo & .po file based on the .pot file, which is located in the `/language` folder of this plugin.
